templates/impala/base.html

```diff
@@ -56,12 +56,11 @@
         data-collect-timings="{{ url('amo.timing.record') }}:{{ collect_timings_percent }}"
         {% block bodyattrs %}{% endblock %}>
 
-    <div id="tabzilla-wrapper">
+    <div id="main-wrapper">
       <div id="page" class="c">
```

Comment:
The only things that are in common are
1/ the tabzilla code
2/ the `<nav>`
But they don't have the same direct parent, so I can't factor them both away.
I factored the three lines of tabzilla code in a separate template, but I won't touch the nav for now, as it's not related to this PR. I'll open a new one (#627)